kibana-reports/server/routes/index.ts

```diff
@@ -13,15 +13,15 @@
  * permissions and limitations under the License.
  */
 
-import registerVisualReportRoute from './visualReport';
+import registerReportRoute from './report';
```

Comment:
What's the purpose for changing from `VisualReport` to just `Report` before merging in with `DataReport`?

Reply:
Not only the file named changed, the logic inside has been updated as well. It's not only for visual report now. Will merge data report later in refactor PR.